src/main.ts

```diff
@@ -0,0 +1,9 @@
+import { arweaveUploader } from './arweaveUploader';
```

Comment:
please move to src/index.ts

Reply:
it's in a separate file because of future testing. When I test the code I don't want to start the health server. So for testing I will only require this file but not index.ts